elastic/eui
# [Storybook] Create Stories for `EuiSideNav`

PR 1/2 for #7176

# Summary
This story creates a Storybook playground and example for `EuiSideNav`. The playground is a kitchen sink example meant to demonstrate the nesting capability, mobile view, and truncation for `EuiSideNav`. The `Header` story is meant to provide a basic playground dedicated to `EuiSideNavHeaderProps`.

# QA
- `gh pr checkout 7174`
-  `yarn && yarn storybook`
- Visit http://localhost:6006/?path=/story/euisidenav--playground to view the playground
   - Check that the `truncation`, `isOpenOnMobile`, and `mobileBreakpoints` change as expected
- Visit http://localhost:6006/?path=/story/euisidenav--side-nav-header to view the Side Nav Header example
